.github/workflows/update-screenshots.yml

```
@@ -49,17 +49,25 @@ jobs:
           DEBIAN_FRONTEND=noninteractive sudo apt-get install --no-install-recommends -yq \
             libasound2-dev libudev-dev libxkbcommon-x11-0;
 
-      - name: install xvfb, llvmpipe and lavapipe
+      - name: Install xvfb, llvmpipe and lavapipe
         run: |
           sudo apt-get update -y -qq
           sudo add-apt-repository ppa:kisak/turtle -y
           sudo apt-get update
           sudo apt install -y xvfb libegl1-mesa libgl1-mesa-dri libxcb-xfixes0-dev mesa-vulkan-drivers
 
+      - name: Install oxipng
+        run: |
+          cargo install oxipng
+
       - name: Take Screenshots
         run: |
           xvfb-run cargo run -p example-showcase -- --page ${{ matrix.page }} --per-page ${{ env.PER_PAGE }} run --screenshot --in-ci
 
+      - name: Optimize PNGs
+        run: |
+          oxipng -o 6 -s -r screenshots
```

```suggestion
          oxipng --opt max --strip safe --recursive screenshots
```

I would prefer to use the long form of the options. It's more clear for people unfamiliar reading over the workflow, and it increases the likelihood of typos being caught. (`-o max` is the same as `-o 6`, but it is more clear on what behavior we want from the optimizer.)